src/libmongoc/src/mongoc/mongoc-cmd.c

```diff
@@ -785,6 +785,29 @@ _is_retryable_read (const mongoc_cmd_parts_t *parts,
 }
 
 
+static bool
+_txn_in_progress (mongoc_cmd_parts_t *parts)
+{
+   mongoc_client_session_t *cs;
+
+   cs = parts->prohibit_lsid ? NULL : parts->assembled.session;
+   if (!cs) {
+      return false;
+   }
+
+   if (_mongoc_client_session_txn_in_progress (cs)) {
+      return true;
+   }
+
+   /* commitTransaction counts, too. */
+   if (parts->assembled.is_txn_finish) {
```

Comment:
`is_txn_finish` is also true for `abortTransaction`. I think that's right. It might not be exercised in the transaction-handing.json test.

cc @alcaeus Can you confirm `abortTransaction` should skip adding server API options?

Reply:
Yes, `abortTransaction` should not include versioned API options, as it is still bound to the API version set when the transaction was started.

Reply:
Great!  I'll just change the comment, then.